etc/db-config.yaml

```diff
@@ -281,6 +281,15 @@
             public: true
             description: If set, enable teams and jury to send source code to this command. See admin manual for allowed arguments.
             docdescription: See :ref:`printing` for more information.
+        -   name: event_feed_format
+            type: int
+            default_value: 0
+            public: false
+            description: "Format of the event feed to use"
+            options:
+                0: New format in use since the `2022-07` version # TODO: when spec is finalized, update version
+                1: Legacy format in use until the `2020-03` version
```

Comment:
I think it makes more sense to renumber them from `1` and incrementing, so that if we add a new version, it becomes `2` instead of having to do other weird numbering schemes.

Reply:
I propose to than swap the 0 & 1, datasource is already 0-indexed so I prefer to not have this configuration to be 1-indexed.

Reply:
I’m fine with either

Reply:
I think the more important question is whether the `0` value is a good default. Because, if there is no good default, then `0` can mean undefined. I guess in this case `0 = legacy format` seems reasonable to me, so fine with this. But in general, I wouldn't always stick with zero-based indexing, just because we've done it once before.